notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -76,12 +76,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import urllib.request
</pre><p>do we really need utils here if you need to download only one image?</p><p> </p><p>this notebook has dependencies on diffusers lib, which has own utility for downloading images</p><p> </p><p>from diffusers.utils import load_image</p><p> </p><p>image = load_image(image_url)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>For gradio inference, all used files should be in the same directory as the notebook. <code>diffusers.utils.load_image</code> function does not download the file itself, but reads the PIL image from HTTP response bytes. A simpler solution could be using <code>urllib.request</code> library.</p>